modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -103,7 +117,11 @@ load balancers and routers to manage the traffic.
 one IP address pool. If you need to access the services from an external network,
 configure load balancers and routers to manage the traffic.
 <10> You must set the platform to `none`. You cannot provide additional platform
-configuration variables for bare metal infrastructure.
+configuration variables for
+ifndef::ibm-z,ibm-power[bare metal]
+ifdef::ibm-z[IBM Z]
+ifdef::ibm-power[IBM Power]
```

Comment:
This (and the heading for the module) still say bare metal for IBM Power. Looks like the top of the page only sets the ibm-power attribute for the restricted network version.

Reply:
Yes, fixed.